src/icons/ChevronDown/ChevronDown.native.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Svg, { Path } from 'react-native-svg';
+import { getThemeColors } from '../../_helpers/theme';
+
+export default function ChevronDown({ width, height, fill }) {
+  return (
+    <Svg
+      width={width}
+      height={height}
+      viewBox="0 0 24 24"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <Path
+        fillRule="evenodd"
+        clipRule="evenodd"
+        d="M5.29289 8.29289C5.68342 7.90237 6.31658 7.90237 6.70711 8.29289L12 13.5858L17.2929 8.29289C17.6834 7.90237 18.3166 7.90237 18.7071 8.29289C19.0976 8.68342 19.0976 9.31658 18.7071 9.70711L12.7071 15.7071C12.3166 16.0976 11.6834 16.0976 11.2929 15.7071L5.29289 9.70711C4.90237 9.31658 4.90237 8.68342 5.29289 8.29289Z"
+        fill={fill}
+      />
+    </Svg>
+  );
+}
+
+ChevronDown.propTypes = {
+  width: PropTypes.number,
+  height: PropTypes.number,
+  fill: PropTypes.oneOf(getThemeColors()),
```

Comment:
Should we accept theme color keys here instead? Like primary.800 & then resolve the color on the component? Not sure about this. Lmk what you think.

Reply:
Yes why can't we do this?

Reply:
`fill` comes from `Icon` component which is resolved from the theme it self. All the components in the `/icons` folder (namely `checkboxFilled`, `checkboxOutlined`) accept the derived values. So accepting a theme color like "primary.900" here, then again making use of theme context to derive the value doesn't make sense to me.
@deorukhkarchaitanya

Reply:
Yeah. These components are in a way ```assests```. Passing theme here would be odd.

Reply:
It's like asking designers to give us Svgs with theme aliases